docusaurus/docs-internals/editor-controls/data-controls/fileUpload.md

```diff
@@ -0,0 +1,177 @@
+| `helper?.content`           | `string`                                                                                                                                                                                   |    -    | If provided, an icon is displayed next to the label. When hovering over this icon, a tooltip with additional information appears.                                                                                                                                                                                                                                                                                                                                                                                                               |
+| `helper?.position`          | `"top-start"` \| `"top"` \| `"top-end"` \| `"right-start"` \| `"right"` \| `"right-end"` \| `"bottom-end"` \| `"bottom"` \| `"bottom-start"` \| `"left-end"` \| `"left"` \| `"left-start"` | `"top"` | Specifies the position of the tooltip relative to the helper icon.                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| `config?.allowedExtensions` | `Array<string>`                                                                                                                                                                            |    -    | An array of strings that specifies the file extensions allowed for upload. This configuration ensures that only files matching the specified extensions can be uploaded, enhancing security and maintaining control over the types of files that users can upload. <br/><br/>For example: <br/>**`["video/*", ".jpg", ".png]`**<br/><br/>This setup allows for the upload of any video format, as well as JPEG, JPG and PNG image formats.<br/><br/> If `config?.allowedExtensions` not provided, then user allowed to upload any type of file. |
+
+### Return value
+
+The return value is string that, consist from `id` and `name` of the file.
```

Comment:
The return value is a string that contains the file's ID and name, separated by "|||".

P.S. mark "string" as code